tex/chapters/03-Theory.tex

```diff
@@ -3,88 +3,152 @@
+        C & D & C & C \\
+        D & D & C & D \\
+          & C & D & D \\
+        \bottomrule
+    \end{tabular}
+    \caption{The different responses of Pavlov, Pavlov's Dual and Flipped Pavlov}
+    \label{tab:strat-dual-flip}
 \end{table}
 
 The subtle difference between Dual and Flip can be highlighted further by inspecting each row individually.
 
-\underline{Row 1} - Pavlov always plays `C' on the first go.
-Flip will change this to `D'.
-Dual knows that Pavlov always plays `C' regardless of the opponents history and so swaps to `D'.
+\underline{Row 1} - Pavlov always plays $C$ on the first go.
+Flip will change this to $D$.
+Dual knows that Pavlov always plays $C$ regardless of the opponents history and so swaps to $D$.
 
-\underline{Row 2} - In the previous round for Pavlov the strategies played $(C, C)$, and so Pavlov plays `C' again.
-For Flip, the preceding interaction was $(D, C)$, in this instance Pavlov would play `D' again, so this gets flipped to `C'.
+\underline{Row 2} - In the previous round for Pavlov the strategies played $(C, C)$, and so Pavlov plays $C$ again.
+For Flip, the preceding interaction was $(D, C)$, in this instance Pavlov would play $D$ again, so this gets flipped to $C$.
 The previous turn for Dual was $(D, C)$ so it infers that Pavlov had $(C, C)$.
-It knows that Pavlov would play `C' and so plays `D'.
+It knows that Pavlov would play $C$ and so plays $D$.
 
-\underline{Row 3} - In the previous round for Pavlov the strategies played $(C, D)$, and so Pavlov would change to play `D'.
-For Flip, the preceding interaction was $(C, D)$, in this instance Pavlov would change to `D', so this gets flipped to play `C' again.
+\underline{Row 3} - In the previous round for Pavlov the strategies played $(C, D)$, and so Pavlov would change to play $D$.
+For Flip, the preceding interaction was $(C, D)$, in this instance Pavlov would change to $D$, so this gets flipped to play $C$ again.
 The previous turn for Dual was $(D, D)$ so it infers that Pavlov had $(C, D)$.
-It knows that Pavlov would play `D' in this instance and so plays `C'.
+It knows that Pavlov would play $D$ in this instance and so plays $C$.
 
 By following this procedure we can see that the Dual responses and Pavlov's responses are always opposite.
 This is achieved by the Dual inferring what the original strategy must have played and then reacting appropriately.
 In the above example, Pavlov's memory is only one turn deep, so the Dual only has to infer the preceding turn, however, for more complex strategies, it may need to infer the entire history of the original strategy in order to proceed correctly.
 
+% can now extend the definition of the dual to any strategy to a finite horizon, move algorithm
+% the definition is strategy obtained by applying this algorithm
+% example - table for majority dual, similar to pavlov
+% extended dual is still the dual
+% in implementation chapter state that we implement the extended dual
+
+
+
+
```

Comment:
Needs more of an introduction:

- Now that we have these tools we can define  ... which allows for ...